components/src/core/components/CardTable/Cell/DefaultCellContainer.vue

```diff
@@ -10,6 +10,7 @@
       :item="items[header.name]"
       :header="header"
       :rowItem="items"
+      :modelValue="items.selectedStage"
```

Comment:
if `modelValue` is used, you will need to emit `modeValue` change event for `v-model` to work

Reply:
fixed